src/components/ebay-badge/index.js

```diff
@@ -2,9 +2,9 @@ const processHtmlAttributes = require('../../common/html-attributes');
 const template = require('./template.marko');
 
 function getInitialState(input) {
-    const number = Number(input.number);
+    const number = Math.round(input.number);
     const imgRole = (input.type !== 'menu' && input.type !== 'icon');
-    const showBadge = !isNaN(number);
+    const showBadge = !isNaN(number) && number >= 0;
```

Comment:
Technically we could change this to just `const showBadge = number >= 0;` since `NaN` is not greater than, less than or equal to any number.

It might be more clear to leave it as is though, curious what others think?

Reply:
We could certainly simplify it. Simplicity usually wins the day.

Reply:
I have changed this logic as per @DylanPiercey' s comments